Documentation/artifact-format-v3.md

```diff
@@ -383,9 +381,8 @@ These files and attributes are allowed:
   ```
 
 At the moment ONLY a `type-info` file is allowed which can contain only
-`artifact_depends` with one field: `rootfs_image_checksum` parameters and type
-of the payload.
-
+`artifact_depends` where each key is a string, and each value is either a
+string, or a list of strings.
```

Comment:
Actually, as planned it can take on any value that the original `type-info` file can have (this was written quite a long time ago). However, augmented artifacts are rejected by the client currently, so it's not actually in use. Until we start using it, I would remove this whole paragraph. No need to mention any restrictions at this point.